src/client/Microsoft.Identity.Client/TelemetryCore/TelemetryConstants.cs

```diff
@@ -44,6 +44,8 @@ internal static class TelemetryConstants
         public const string Platform = "Platform";
         public const string ApiId = "ApiId";
         public const string IsProactiveRefresh = "IsProactiveRefresh";
+        public const string CallerSdkId = "CallerSdkId";
```

Comment:
Id or name?

Reply:
It will be an id like `1001` or something.